frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card.tsx

```diff
@@ -59,18 +59,18 @@ const StorageEfficiencyCard: React.FC<DashboardItemProps> = ({
 
   const compressionStats = () => {
     const capacityRatio = Number(ratio);
-    return capacityRatio === 0 ? '1:1' : `${Math.round(capacityRatio)}:1`;
+    return `${Math.round(capacityRatio)}:1`;
   };
 
   const savingStats = () => {
     const savingsValue = Number(saved);
     const savedBytes = humanizeBinaryBytes(savingsValue).string;
-    return savingsValue === 0 ? 'No Savings' : savedBytes;
+    return savedBytes;
   };
 
   const compressionRatioProps = {
     isLoading: !poolCapacityRatioResult,
-    error: !!poolCapacityRatioResultError || !ratio,
+    error: !!poolCapacityRatioResultError || !ratio || Number(ratio) === 0,
```

Comment:
Instead you might want to add this as a condition in `ItemsBody` alongwith `error` .
Since 0 stats is not technically an error.

Reply:
Every Item can have its own reason to have `error = true`. It's true here for both the items, but we plan to have more items in the card in future releases, so IMO we shouldn't generalise it. wdyt ?

Reply:
The same piece of check is required in noobaa reduction card.
It will work for all four use cases currently.